frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -133,16 +167,32 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
           onChange={handleModeChange}
         />
       </div>
-      {mode === MODES.INTERNAL && <CreateInternalCluster match={match} mode={mode} />}
-      {mode === MODES.EXTERNAL && (
+      {persistMode === MODES.INTERNAL && (
+        <CreateInternalCluster
+          getStep={getStep}
+          getParamString={getParamString}
+          getIndex={getIndex}
+          getAnchor={getAnchor}
```

Comment:
I'll rather pass them in a single object than having multiple props

Reply:
Sure, sending a patch!